themes/digital.gov/layouts/communities/single.html

```diff
@@ -33,139 +32,128 @@ <h1>{{ .Title | markdownify }} Community</h1>
-                    {{- end -}}
-
-                    {{/* Listserv Platforms with form as sign-up option */}}
-                    {{- if and (eq $e.platform "listserv") ( $e.subscribe_form ) -}}
-                      {{- partial "core/community-join-with-form.html" (dict "context" . "e" $e) -}}
-                    {{- end -}}
-
-                    {{/* Google Group platforms */}}
-                    {{- if and (eq $e.platform "google-group") ( $e.subscribe_form ) -}}
-                      {{- partial "core/community-join-google-group.html" (dict "context" . "e" $e) -}}
-                    {{- end -}}
-
-                    {{/* Slack platforms */}}
-                    {{- if and (eq $e.platform "slack") ( $e.subscribe_form ) -}}
-                      {{- partial "core/community-join-slack.html" (dict "context" . "e" $e) -}}
-                    {{- end -}}
-
-
-                    {{/*
-                      Community Members
-                      - These are updated manually in the front matter by going to the backend of the listserv platform and entering in the values one by one.
-                    */}}
-                    {{- if $e.members -}}
-                      <p class="members">
-                        <svg class="usa-icon dg-icon dg-icon--standard margin-bottom-05" aria-hidden="true" focusable="false">
-                          <use xlink:href="{{ "uswds/img/sprite.svg#person" | relURL }}"></use>
-                        </svg>
-                        {{ $e.members }} members
-                        {{- if $e.emails_per_week }}
-                        <svg class="usa-icon dg-icon dg-icon--standard margin-bottom-05" aria-hidden="true" focusable="false">
-                          <use xlink:href="{{ "uswds/img/sprite.svg#mail" | relURL }}"></use>
-                        </svg>
-                        Avg. {{ $e.emails_per_week }} emails per week{{- end -}}
-                      </p>
-                    {{- end -}}
+                      {{/* Google Group platforms */}}
+                      {{- if and (eq $e.platform "google-group") ( $e.subscribe_form ) -}}
+                        {{- partial "core/community-join-google-group.html" (dict "context" . "e" $e) -}}
+                      {{- end -}}
 
+                      {{/* Slack platforms */}}
+                      {{- if and (eq $e.platform "slack") ( $e.subscribe_form ) -}}
+                        {{- partial "core/community-join-slack.html" (dict "context" . "e" $e) -}}
+                      {{- end -}}
 
+                    </div>
                   </div>
-                </div>
-              {{- end -}}
+                {{- end -}}
 
-              {{- if .Params.authors -}}
-              <div class="community_managers">
-                <h3>Community Managers</h3>
-                {{- partial "core/get_authors.html" . -}}
-              </div>
+              </section>
               {{- end -}}
+              <div class="usa-prose">
```

Comment:
It looks like there's already a `usa-prose` class on the `<main>` element. This div might not be necessary.

Reply:
I've kept the div but removed the class since it inherits from the `<main>` element.